packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -2,6 +2,8 @@
     background: var(--general-main-2, #ffffff);
     border-radius: 0.8rem;
     animation: fadein 0.6s ease backwards;
+    max-height: calc(100vh - 10rem);
+    overflow-y: auto;
```

Comment:
we already have a `max-height` for desktop is there gonna be any conflict between the two?

Reply:
It's set to 100vh, which is always related to the viewport height. Therefore, I don't think it will cause any conflicts.